cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/api/InvoiceApi.java

```diff
@@ -67,6 +67,13 @@
     @PATCH( "invoice/{id}/confirmPaypalPayment" )
     Call<Void> payWithPaypal( @Path( "id" ) int id, @Body Map<String, String> input );
 
+    @POST( "invoice/{id}/pay/stripe" )
+    Call<Void> payWithStripe( @Path( "id" ) int id );
+
+    /**
+     * @deprecated by {@link #payWithStripe(int)}
+     */
+    @Deprecated
     @POST( "invoice/{id}/pay/stripe" )
     Call<Void> payWithStripe( @Path( "id" ) int id, @Body CardDataDTO input );
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)